Integrations/Shodan/Shodan.yml

```diff
@@ -0,0 +1,235 @@
+description: search engine for Internet-connected devices
+configuration:
+- display: Api Key
+  name: apiKey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Base url to Shodan API
+  name: baseUrl
+  defaultvalue: https://api.shodan.io
+  type: 0
+  required: true
+- display: Trust self-signed certificate (insecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: '-'
+  type: python
+  commands:
+  - name: search
+    arguments:
+    - name: query
+      required: true
+      default: true
+      description: 'Shodan search query. The provided string is used to search the
+        database of banners in Shodan, with the additional option to provide filters
+        inside the search query using a "filter:value" format. For example, the following
+        search query would find Apache webservers located in Germany: "apache country:DE"'
+    - name: facets
+      description: A comma-separated list of properties to get summary information
+        on. Property names can also be in the format of "property:count", where "count"
+        is the number of facets that will be returned for a property (i.e. "country:100"
+        to get the top 100 countries for a search query)
+    - name: page
+      description: Result page number to be fetched. Each page contains up to 100 results.
+      defaultValue: "1"
+    description: Search Shodan using the same query syntax as the website and use
+      facets to get summary information for different properties.
```

Comment:
no outputs? @Arsenikr

Reply:
will work on context for the search command